airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -189,6 +190,7 @@ def patch_dag_run(
             status.HTTP_404_NOT_FOUND,
         ]
     ),
+    dependencies=[Depends(requires_access_asset(method="GET"))],
```

Comment:
Add a todo, we need also `@security.requires_access_dag("GET", DagAccessEntity.RUN)`

Reply:
I will add this when I am working on dag_run. which I will be picking next

Reply:
Yes, I would still be in favor of at least adding a `TODO` comment so we do not forget about it.